Modules/Core/Common/test/itkImageIORegionGTest.cxx

```diff
@@ -0,0 +1,186 @@
+/*=========================================================================
+ *
+ *  Copyright Insight Software Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+// First include the header file to be tested:
+#include "itkImageIORegion.h"
+#include <gtest/gtest.h>
+#include <random>
+#include <type_traits> // For is_default_constructible, is_copy_constructible, etc.
+
+namespace
+{
+template <typename T>
+constexpr bool                               IsDefaultConstructibleCopyableNoThrowMovableAndDestructible =
```

Comment:
What is this aligned with? Does `clang-format` produce this?

Reply:
@dzenanz Well observed! But honestly, I swear I only typed a single space... `clang-format` alignment automation messed it up quite badly :astonished: in this particular case, I guess it's just a clang-format bug. But of course, I'm still in favor of entirely reconsidering the auto-align option https://discourse.itk.org/t/update-coding-style-for-itk/2055/10